config/routes.rb

```diff
@@ -20,7 +20,12 @@
   resources :conferences
   resources :attendances
   resources :contributors, only: :index
+
   resources :status_updates, only: :show
+  namespace :status_updates do
```

Comment:
I think the `StatusUpdateCommentsController` is a very good idea. This probably was a leftover from the path_parents-path I travelled ;-) I will have a look tomorrow.